hasher-matcher-actioner/hmalib/common/tests/test_s3_adapters.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from unittest import TestCase
+
+from hmalib.common.s3_adapters import ThreatUpdateS3Store, KNOWN_SIGNAL_TYPES
+
+
+class S3AdaptersTestCase(TestCase):
+    def test_key_indicator_type_mapping(self):
+        for signal_type in KNOWN_SIGNAL_TYPES:
+            store = ThreatUpdateS3Store(
```

Comment:
nit: I like an overuse of keyword args in tests to I know what 'does-not-matter'